src/components/LoginForm/LoginForm.jsx

```diff
@@ -66,9 +78,14 @@ return(
     </div>
     <br/>
     <Button text={"Login"} btnClass = "btn-info " onClick={handleSubmit}/>
-  
    
+    <p className="error-message">{errorMessage}</p>
```

Comment:
This works really well.
It might make more sense to only have the errorMessage show if the form is not set as invalid, and only show the "please check your username..." if the form is invalid (and have the formIsInvalid message above the sub messages):

```   
    <p className={formIsInvalid ? "hidden":"error-message"}>{errorMessage}</p>
    <p>{formIsInvalid}</p>
    <sub className={formIsInvalid ? "" : "hidden"}>Please check your username and password.</sub>
    <sub className={formIsInvalid ? "" : "hidden"}>* Username and password are case sensitive.</sub>```